setup.py

```diff
@@ -349,7 +293,6 @@ def get_outputs(self):
 	platforms			= version.platforms,
 	classifiers			= version.classifiers,
 	cmdclass			= {'build_ext': build_ext},
-	ext_modules			= [Extension(m.replace('/', '.'), sources=[]) for m in extension_module_names],
```

Comment:
Why is this changed?

Reply:
Thought it will not be needed anymore, but it is for sdist. Will post an update.